compiler/compiler/src/main/java/org/robovm/compiler/target/AbstractTarget.java

```diff
@@ -530,14 +538,15 @@ protected void stripArchive(Path path, File output) throws IOException {
                 try {
                     archive = new ZipFile(path.getFile());
                     Enumeration<? extends ZipEntry> entries = archive.entries();
-                    while (entries.hasMoreElements()) {
+                    ENTRIES: while (entries.hasMoreElements()) {
                         ZipEntry entry = entries.nextElement();
                         if (entry.getName().toLowerCase().endsWith(".class")) {
                             continue;
                         }
```

Comment:
Couldn't we add this also to the default patterns, like ".*\.class", or causes the backslash problems in the xml (it shouldn't)? Then we could merge this PR, thanks for your work!